node/tests/TestUtilities.ts

```diff
@@ -367,6 +367,10 @@ export async function transactionTest(
     args.push("OK");
     baseTransaction.exists([key10]);
     args.push(1);
+    baseTransaction.renamenx(key10, key9);
+    args.push(true);
+    baseTransaction.exists([key9]);
```

Comment:
if we change this to: 
```
baseTransaction.exists([key9, key10]);
```
I think we get the same result and it would also indicate that key10 was renamed.